images/Containerfile.pipeline

```diff
@@ -1,3 +1,32 @@
+# This is the builder image, see the next `FROM` directive for the beginning of the pipeline image:
+FROM registry.access.redhat.com/ubi9/ubi:9.1.0-1750 AS builder
+
+# Install the Go toolset:
+RUN \
+    curl --location --retry 5 --output /tmp/tarball https://go.dev/dl/go1.20.linux-amd64.tar.gz && \
+    echo 5a9ebcc65c1cce56e0d2dc616aff4c4cedcfbda8cc6f0288cc08cda3b18dcbf1 /tmp/tarball | sha256sum --check && \
+    tar --directory /usr/local --extract --file /tmp/tarball && \
+    rm /tmp/tarball
```

Comment:
In the assisted-service `Dockerfile` we do this:

```
COPY --from=registry.ci.openshift.org/openshift/release:golang-1.18 /usr/local/go /usr/local/go
```

I wonder if we should do the same here to make sure we use the same go version everywhere. Maybe not copy it but use that image as the base-image. Thoughts?

Reply:
It is a bit sad to have to downgrade the version of Go. Currently the project uses 1.20 but there is no OpenShift release image for that. Would it be OK to downgrade to 1.19? According to the Go policy 1.18 is already out of support. The OpenShift release image does exists for 1.19.

Reply:
> It is a bit sad to have to downgrade the version of Go. Currently the project uses 1.20 but there is no OpenShift release image for that. Would it be OK to downgrade to 1.19? According to the Go policy 1.18 is already out of support. The OpenShift release image does exists for 1.19.

Yeah, let's use `1.19` then unless we really need `1.20`

Reply:
OK, updated to use the 1.19 release image.